cluster/network.go

```diff
@@ -365,12 +365,12 @@ func (c *Cluster) runServicePortChecks(ctx context.Context) error {
 			return err
 		}
 	}
-	// check all -> etcd connectivity
+	// check control -> etcd connectivity
 	log.Infof(ctx, "[network] Running control plane -> etcd port checks")
 	for _, host := range c.ControlPlaneHosts {
 		runHost := host
 		errgrp.Go(func() error {
-			return checkPlaneTCPPortsFromHost(ctx, runHost, EtcdPortList, c.EtcdHosts, c.SystemImages.Alpine, c.PrivateRegistriesMap)
+			return checkPlaneTCPPortsFromHost(ctx, runHost, []string{EtcdPort1}, c.EtcdHosts, c.SystemImages.Alpine, c.PrivateRegistriesMap)
```

Comment:
Keep the list even if it's a single item.. more readable and can be referenced later if needed.